### PR TITLE
Added the ability to restore Facebook session without the need for user interaction

### PR DESCRIPTION
**Pull request by ArtLapitski**
In our app we want to provide our users an autologin feature. Facebook is just one of the login methods the app supports, so when the app starts we need a way to check whether user is already logged in using his facebook account. Currently there is no way to perform such check without potentially invoking the login dialog.
I added `SessionLoginBehavior::NoFallbackToUi` member which forces `LoginAsync()` to restore the active session or return `FBError` if there is no valid access token.
Supersedes #94

### CODE REVIEW

File: winsdkfb/winsdkfb/winsdkfb.Shared/FacebookSession.cpp [hunk 1]
```diff
@@ -1043,6 +1047,76 @@ task<FBResult^> FBSession::TryLoginViaWebAuthBroker(
...
+        .then([=](FBResult^ oauthResult) -> task<FBResult^>
+    {
+        task<FBResult^> graphTask = create_task([]() -> FBResult^
+        {
+            return nullptr;
+        });
+
+        if (oauthResult && oauthResult->Succeeded)
+        {
+            Facebook::FBAccessTokenData^ tokenData =
```
Comment (adiviness) [inline winsdkfb/winsdkfb/winsdkfb.Shared/FacebookSession.cpp]: `Facebook` namespace declaration should be removed, not required and causes compile error on `develop` branch with #71.

File: winsdkfb/winsdkfb/winsdkfb.Shared/FacebookSession.cpp [hunk 2]
```diff
@@ -1043,6 +1047,76 @@ task<FBResult^> FBSession::TryLoginViaWebAuthBroker(
...
+                static_cast<Facebook::FBAccessTokenData^>(oauthResult->Object);
```
Comment (adiviness) [inline winsdkfb/winsdkfb/winsdkfb.Shared/FacebookSession.cpp]: Same as with previous line.

**Comment by ArtLapitski** [comment]
Oops, I definitely had to check the compilation after rebase of my branch. Fixed!

**Comment by adiviness** [comment]
Thanks! Sorry it's taken so long for me to get back to this, a few changes have taken place in the `develop` branch that would now create merge conflicts for this pull request. Additionally, we have renamed the enum values in `SessionLoginBehavior` to better reflect their intentions, if you could update your new one to something like `Silent` that would be great and I could then merge it in.

**Comment by ArtLapitski** [comment]
Hi! Sorry for the late response, I was away for a while. Branch is now updated to the latest `develop`. I called the new login behavior `Silent` just like you proposed.

**Comment by CristiW** [comment]
Hello. This will be a good addition because adds the option to re use the cached access token.

**Pull request merged**